modules/food-menu/types.js

```diff
@@ -27,6 +27,7 @@ export type MenuItemType = {
 				well_being: string,
 				well_being_image: string,
 		  },
+	nutrition_details: {} | NutritionDetailContainer,
```

Comment:
Is this still true? If we have an item with no nutritional details, will it just be a blank object, or will it be `null` or something?

Reply:
An empty `nutrition_details` element in the JSON looks like this

```json
"nutrition_details": {},
```